### PR TITLE
Remove `-i` key in `docker exec` in `bin/resolve.sh`

### PR DESCRIPTION
**Pull request by aprasolova**
Signed-off-by: anastasia prasolova <anastasia@nspcc.ru>

### CODE REVIEW

**Review by EliChin — DISMISSED** [review]

File: services/s3_gate/wallet.json [hunk 1]
```diff
@@ -1 +1,32 @@
-{"version":"3.0","accounts":[{"address":"NUUb82KR2JrVByHs2YSKgtK29gKnF5q6Vt","key":"6PYSPET41jKtqie2cfkqgy7q9ueeucH8bX9Gotm2HwdNEDptj6aBe8dDg8","label":"NeoFS S3 Gate","contract":{"script":"DCEDE7GsOoB24VWn55eyTwtlDMytWUHqWdfP1RoCSosqBr9BVuezJw==","parameters":[{"name":"parameter0","type":"Signature"}],"deployed":false},"lock":false,"isDefault":false}],"scrypt":{"n":16384,"r":8,"p":8},"extra":{"Tokens":null}}
+{
```
Comment (carpawell) [inline services/s3_gate/wallet.json:1]: could you, please, make that change in a separate commit?

Comment (aprasolova) [reply services/s3_gate/wallet.json:1]: Sure, fixed

**Review by carpawell — APPROVED** [review]

**Review by alexvanin — APPROVED** [review]
